packages/runtime-common/realm.ts

```diff
@@ -231,15 +256,34 @@ export class Realm {
   }
 
   constructor(
-    url: string,
-    adapter: RealmAdapter,
-    loader: Loader,
-    indexRunner: IndexRunner,
-    runnerOptsMgr: RunnerOptionsManager,
-    getIndexHTML: () => Promise<string>,
+    {
+      url,
+      adapter,
+      loader,
+      indexRunner,
+      runnerOptsMgr,
+      getIndexHTML,
+      matrix,
+      realmSecretSeed = uuidV4(),
```

Comment:
I think it would be safer to remove this default value and make the value mandatory. I was imagining a case where someone would forget to provide `realmSecretSeed` when starting a realm and then the existing JWTs in the wild would stop working because the seed would not match.

Reply:
Well the issue is that the secret seed allows the tokens to stay stable between server restarts. With that default value we still have a secret, but it changes with each server restart meaning the token becomes invalid (the client will just get a new one on startup). It’s pretty innocuous really, but we can make this a required value.

Reply:
fixed